dune-project

```diff
@@ -0,0 +1,4 @@
+(lang dune 2.7)
+(using coq 0.2)
+(name coq-dpdgraph)
+(cram enable)
```

Comment:
```suggestion
(cram enable)

```

Reply:
Not sure what the extra newline here would add, the convention in Coq-community is usually to skip them.